metricbeat/module/mongodb/dbstats/_meta/docs.asciidoc

```diff
@@ -1 +1,44 @@
 This is the `dbstats` metricset of the MongoDB module.
+
+It requires the following privileges:
+
+- https://docs.mongodb.com/manual/reference/privilege-actions/#listDatabases[`listDatabases` action] on https://docs.mongodb.com/manual/reference/resource-document/#cluster-resource[`cluster` resource]
+- for each of the databases, also need https://docs.mongodb.com/manual/reference/privilege-actions/#dbStats[`dbStats` action] on the https://docs.mongodb.com/manual/reference/resource-document/#database-and-or-collection-resource[`database` resource]
+
+You can use the following command in Mongo shell to create the role (make sure you are using the `admin` db by using `db` command in Mongo shell).
+
+["source","js",subs="attributes"]
+----
+db.createRole(
+   {
+     role: "dbstatsRole",
+     privileges: [
+       { resource: { cluster: true }, actions: ["listDatabases"] },
+       // for each of the databases
+       { resource: { db: "db_1", collection: ""}, actions: ["dbStats"]},
+       { resource: { db: "db_2", collection: ""}, actions: ["dbStats"]}
+     ],
+     roles: []
+   }
+)
+----
```

Comment:
As @ChrsMark mentions, I think we can remove the example commands from the metricsets, and just list the required privileges for reference, mentioning that they are included in the `clusterMonitor` role.